kolibri/plugins/coach/assets/src/modules/classSummary/index.js

```diff
@@ -431,14 +431,16 @@ export default {
     SET_CLASS_LESSONS_SIZES(state, sizes = {}) {
       state.lessonsSizes = sizes;
     },
-    SET_CLASS_QUIZZES_SIZES(state, sizes = {}) {
+    SET_CLASS_QUIZZES_SIZES(state, sizes) {
       if (sizes.length > 0) {
-        sizes.forEach(sizeItem => {
+        for (const sizeItem of sizes) {
```

Comment:
Yes - I just did this because for... of.. is generally faster than `forEach` (because of the redefinition of the anonymous function). So this was a flyby cleanup.